poll.go

```diff
@@ -388,10 +388,14 @@ func (h *Headscale) PollNetMapStream(
 			updateRequestsReceivedOnChannel.WithLabelValues(machine.Name, machine.Namespace.Name).
 				Inc()
 			if h.isOutdated(machine) {
+				var lastUpdate time.Time
```

Comment:
oh, good catch...

Reply:
It might have been better off in a separate MR though? I think I patched a number of these while working on getting integration tests to not crash - the IPv6 branch was a lot more likely trigger them.

Reply:
Yes, but I am keen to get them in, so we can live with that for now :)